### PR TITLE
Add description for plugin

### PR DESCRIPTION
**Pull request by sandysaille**
Add more release files, icon, and banner for this plugin to push to Wordpress.org

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: php
+
+php:
+    - 5.3
+    - 5.4
+    - 5.5
+    - 5.6
+    - 7.0
+    - hhvm
```
Comment (zkan) [inline .travis.yml]: We don't need HHVM anymore, I think. Let's remove it.

File: readme.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+== Installation ==
+1. Upload the plugin files to the `/wp-content/plugins/plugin-name` directory, or install the plugin through the Wordpress plugin screen directly.
+2. Activate the plugin through the 'Plugins' screen in WordPress
+3. Include the shortcode anywhere on your website: `[gravityform-total-amount form_id="14" field_id="18" stat_date="2017-07-18" end_date="2017-07-20"]` 
+
+
+== Changelog ==
+
+= 1.0.0 =
+* First release!
```
Comment (zkan) [inline readme.txt:38]: We should write a short description here. Maybe a short summary of what this plugin can do?

**Review by zkan — CHANGES_REQUESTED** [review]

**Comment by sandysaille** [comment]
@zkan Please review again ka.

**Comment by sandysaille** [comment]
@zkan Please review again ka _/\_

**Comment by zkan** [comment]
@sandysaille I mean the change log section ja, not the description. Please revert the description back. Sorry! >_<

**Comment by sandysaille** [comment]
@zkan Oopss sorry ka, fixed it. Please review again ka.

**Review by zkan — APPROVED** [review]

**Pull request merged**